anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -182,14 +197,18 @@ def toggle_group_enabled(feed, group, enabled):
 
 @authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
 def delete_feed(feed):
-    if feed == GRYPE_DB_FEED_NAME:
+    provider = get_vulnerabilities_provider()
+    internal_feed_name = provider.display_mapper.get_internal_name(feed)
+    if isinstance(provider, GrypeProvider) and internal_feed_name == GRYPE_DB_FEED_NAME:
         raise HTTPNotImplementedError(
-            message="Deleting the grypedb feed is not yet supported.",
+            message="Deleting the {} feed with the grype vulnerability provider enabled is not yet supported.".format(
+                feed
+            ),
             detail={},
         )
     session = db.get_session()
     try:
-        f = db.lookup_feed(db_session=session, feed_name=feed)
+        f = db.lookup_feed(db_session=session, feed_name=internal_feed_name)
```

Comment:
Same, please rename the single character variable.

Reply:
done